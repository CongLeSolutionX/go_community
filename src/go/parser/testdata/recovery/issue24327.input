package issue24327

func f() {
}

func _() {
	f(, /* ERROR "expected operand, found ','" */ )
}
