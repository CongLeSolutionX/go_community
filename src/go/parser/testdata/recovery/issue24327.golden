package issue24327

func f() {
}

func _() {
	f(BadExpr)
}
